cloudsql/mysql/database-sql/cloudsql_test.go

```diff
@@ -22,20 +22,55 @@ import (
 	"testing"
 )
 
+type commonTestInfo struct {
+	dbName                 string
+	dbPass                 string
+	dbUser                 string
+	dbPort                 string
+	instanceConnectionName string
+}
+
+func getCommonTestInfo() commonTestInfo {
```

Comment:
```suggestion
type testInfo struct {
	dbName                 string
	dbPass                 string
	dbUser                 string
	dbPort                 string
	instanceConnectionName string
}

var commonTestInfo = testInfo {
```

No need for the function. Also means we can delete the `_commonTestInfo` var. In general, names should not start with "_" in Go. See https://golang.org/doc/effective_go#names.